### ISSUE TITLE
Content of intro.js Steps dialog is not announced when using assistive technologies like Voice Over

### ISSUE THREAD

**Issue by theindianotaku** [usablica/intro.js#1017]
## Description
When intro.js steps is triggered with a screen reader active, the content of the dialog is not announced.

## Expected Behavior
A dialog when opened, should announce the title and/or the message and focus on the first focusable element in the dialog. 

## Actual Behavior
When used with a screen reader (Voice Over with Safari), the screen reader doesn't announce anything.
A dialog should have `aria-label` or `aria-labelledby` that refers to a visible dialog title and optionally can have `aria-describedby` to provide the message of the dialog.

## Example
Open the demo in the main page of intro.js in a browser with any screen reader active

## Environment
- Voice Over - Safari 14.0 - macOS Catalina
- Using [intro.js-react](https://github.com/interviewstreet/intro.js-react) module in a React app

**Comment by ad1992** [comment]
Any update on this?
